src/client/containers/Login.js

```diff
@@ -0,0 +1,13 @@
+import { connect } from 'react-redux';
+import { push } from 'react-router-redux';
+import Login from '../components/Login';
+
+const mapStateToProps = ({ user }) => ({ user });
+
+const mapDispatchToProps = (dispatch) => ({
+  onSignIn: () => {
+    dispatch(push('/dashboard'));
```

Comment:
sweet!... i didn't know there was a `push` method. i was importing `browserHistory` to do a push.